assignments/arrays.js

```diff
@@ -63,32 +63,62 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a ${inventory[33]["car_year"]} ${inventory[33]["car_make"]} ${inventory[33]["car_model"]}`);
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory.length;
+console.log(`Car ${lastCar} is a ${inventory[lastCar-1]["car_make"]} ${inventory[lastCar-1]["car_model"]}`);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
-let carModels = [];
-console.log();
+const carModels = [];
+
+for (let i=0; i < inventory.length; i++) {
+    carModels.push(inventory[i]["car_model"]);
+}
+
+console.log(carModels.sort());
 
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
 let carYears = [];
-console.log();
+for (let i=0; i < inventory.length; i++) {
+    carYears.push(inventory[i]["car_year"]);
```

Comment:
`inventory[i]["car_year"] is a valid way of accessing the value of a property. another way to do this though is using dot notation with objects. and it would like this
`inventory[i].car_year` use dot notation first when we know the actual name of the property. in cases where the name of the property is not a valid variable name or the name of the property is unknown then use the bracket notation.

Reply:
changed all bracket notation to dot notation